README.md

```diff
@@ -59,9 +59,15 @@ Final submission should be made to contact@black.ai, with the subject line "ML_C
 
 ## Evaluation
 We are primarily looking to assess the accuracy of detection, speed of inference, and the implementation itself. We will be testing/training both the implementation and model on a machine running 64-bit Ubuntu 16.04 LTS, with a Nvidia GTX 1080 graphics card.  
+What are we looking for:
+Your understanding of parameters and hyperparameters is a mandate with this task. You are ewxpected to back your reasoning for choosing the values for all the hyperparameters.
```

Comment:
You are ~~ewxpected~~ **expected** to **be able to** ~~back~~ **explain** your reasoning for any **changes made** to the architecture or ~~choosing the~~ **choices regarding** ~~the values for all~~ **the setting of** hyperparameters.